relay-event-normalization/src/event.rs

```diff
@@ -31,14 +31,20 @@ use crate::normalize::request;
 use crate::span::tag_extraction::extract_span_tags_from_event;
 use crate::utils::{self, get_event_user_tag, MAX_DURATION_MOBILE_MS};
 use crate::{
-    breakdowns, legacy, mechanism, schema, span, stacktrace, transactions, trimming, user_agent,
-    BreakdownsConfig, DynamicMeasurementsConfig, GeoIpLookup, MaxChars, PerformanceScoreConfig,
-    RawUserAgentInfo, SpanDescriptionRule, TransactionNameConfig,
+    breakdowns, event_error, legacy, mechanism, remove_other, schema, span, stacktrace,
+    transactions, trimming, user_agent, BreakdownsConfig, DynamicMeasurementsConfig, GeoIpLookup,
+    MaxChars, PerformanceScoreConfig, RawUserAgentInfo, SpanDescriptionRule, TransactionNameConfig,
 };
 
 /// Configuration for [`normalize_event`].
 #[derive(Clone, Debug)]
 pub struct NormalizationConfig<'a> {
+    /// Whether it's the last normalization step.
+    ///
+    /// The last normalization run performs a few additional steps that
+    /// shouldn't happen in previous runs, like removing unknown attributes.
```

Comment:
Not quite true, it also needs `remove_other` to be enabled.

I think we can kill it in favor of having specialized bools like `remove_other` and `emit_events`

Reply:
> I think we can kill it in favor of having specialized bools

+1, `is_last_normalize` could be misleading because the final normalization step actually happens in sentry.

Reply:
Killed the config in [59c0c95](https://github.com/getsentry/relay/pull/3097/commits/59c0c950929b63e64998e8ee817a21e7812b8dba).